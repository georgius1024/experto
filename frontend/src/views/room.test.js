import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import View from './room'
describe('Room view', () => {
  test('it matches the snapshot', () => {
    const view = renderer.create(
      <MemoryRouter>
        <View />
      </MemoryRouter>
    )
    expect(view.toJSON()).toMatchSnapshot()
  })
})
