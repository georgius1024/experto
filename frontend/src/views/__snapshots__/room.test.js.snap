// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Room view it matches the snapshot 1`] = `
<p>
  .............loading.............
</p>
`;
